dotcom-rendering/src/server/htmlPageTemplate.ts

```diff
@@ -67,6 +68,7 @@ export const htmlPageTemplate = (props: WebProps | AppProps): string => {
 		renderingTarget,
 		hasPageSkin = false,
 		weAreHiring,
+		alternateLangLinks,
```

Comment:
Wondering if it’s easier to reason about empty arrays than undefined ones…

```suggestion
		alternateLangLinks = [],
```